packages/core/src/html/includePanelProcessor.js

```diff
@@ -255,7 +255,101 @@ function processInclude(node, context, pageSources, variableProcessor, renderMd,
   return childContext;
 }
 
+/**
+ * PreProcesses popovers with the src attribute.
+ * Replaces it with an error node if the specified src is invalid,
+ * or an empty node if the src is invalid but optional.
+ * Else, sets the content attribute of the popover as parsed from the src.
+ */
+function processPopoverSrc(node, context, pageSources, variableProcessor, renderMd, config) {
+  if (!_.has(node.attribs, 'src')) {
+    return context;
+  }
+
+  if (_.isEmpty(node.attribs.src)) {
+    const error = new Error(`Empty src attribute in include in: ${context.cwf}`);
+    logger.error(error);
+    cheerio(node).replaceWith(createErrorNode(node, error));
+    return context;
+  }
+
+  const {
+    isUrl,
+    hash,
+    filePath,
+    actualFilePath,
+  } = _getSrcFlagsAndFilePaths(node, config);
+
+  // No need to process url contents
+  if (isUrl) {
+    const error = new Error('URLs are not allowed in the \'src\' attribute');
+    logger.error(error);
+    cheerio(node).replaceWith(createErrorNode(node, error));
+    return context;
+  }
+
+  const fileExistsNode = _getFileExistsNode(node, context, actualFilePath, pageSources);
+  if (fileExistsNode) {
+    return fileExistsNode;
+  }
+
+  pageSources.staticIncludeSrc.push({
+    from: context.cwf,
+    to: actualFilePath,
+  });
+
+  const {
+    nunjucksProcessed,
+    childContext,
+  } = variableProcessor.renderIncludeFile(actualFilePath, pageSources, node, context, filePath);
+
+  let actualContent = nunjucksProcessed;
+  if (fsUtil.isMarkdownFileExt(path.extname(actualFilePath))) {
+    actualContent = renderMd(actualContent);
+  }
+
+  // Process sources with or without hash, retrieving and appending
+  // the appropriate children to a wrapped include element
+  if (hash) {
+    const $ = cheerio.load(actualContent);
+    actualContent = $(hash).html();
+
+    if (actualContent === null) {
+      actualContent = '';
+
```

Comment:
This part is unnecessary anymore since we will do an early return below

Reply:
I missed this, thanks for the catch!